portal-ui/src/main/java/org/entando/entando/aps/servlet/ControllerServlet.java

```diff
@@ -195,4 +202,42 @@ protected void outputError(RequestContext reqCtx, HttpServletResponse response)
 	private static final String ATTR_APPLICATION_MODEL = ".freemarker.Application";
 	private static final String ATTR_JSP_TAGLIBS_MODEL = ".freemarker.JspTaglibs";
 
+	public static class EntTemplateLoader implements TemplateLoader {
```

Comment:
@eugeniosant i moved from protected to public to make it easy to test, it's ok ?

Reply:
ok